commcare_export/cli.py

```diff
@@ -147,15 +148,20 @@ def main_with_args(args):
     # Reads as excel if it is a file name that looks like excel, otherwise reads as JSON, 
     # falling back to parsing arg directly as JSON, and finally parsing stdin as JSON
     if args.query:
-        query = _get_query(
-            args.query,
-            args.missing_value,
-            writer.supports_multi_table_write,
-        )
+        try:
+            query = _get_query(
+                args.query,
+                args.missing_value,
+                writer.supports_multi_table_write,
+                writer.max_column_length,
+            )
+        except LongFieldsException as e:
+            print(e.message)
+            return 1
```

Comment:
nit: since this is no longer `exit(1)`, it's not obvious what `return 1` means - could make a constant like `EXIT_STATUS_ERROR` or something.

Reply:
961499b